test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2711,8 +2783,12 @@ static int test_sched_queue_priority(const char *shaper_name,
 
 	busy_wait(100 * ODP_TIME_MSEC_IN_NS);
 
-	/* Disable the shaper, so as to get the pkts out quicker. */
-	set_shaper(node_name, shaper_name, 0, 0);
+	/* Disable the shaper, so as to get the pkts out quicker.
+	 * We cannot do this if dynamic shaper update is not supported
+	 * without it, set_shaper() can cause packet drops due to start/stop.
```

Comment:
There is a grammar issue with this sentence. Perhaps it should read: "We cannot do this if dynamic shaper update is not supported. Without dynamic update support set_shaper() can cause packet drops due to start/stop."